frontend/src/components/Help.tsx

```diff
@@ -29,14 +29,14 @@ export default function Help() {
     {
       icon: TbBrandGithub,
       title: 'Release Notes',
-      description: `What's new and improved in Fileglancer version ${versionNo}`,
-      url: `https://github.com/JaneliaSciComp/fileglancer/releases/tag/${versionNo}`
+      description: `What's new and improved in Fileglancer version ${versionQuery.data?.version}`,
+      url: `https://github.com/JaneliaSciComp/fileglancer/releases/tag/${versionQuery.data?.version}`
```

Comment:
There is no loading check to make sure the data has loaded. This could result in a broken link to https://github.com/JaneliaSciComp/fileglancer/releases/tag/ if the user is on a slow connection.